user-guide/modules/ROOT/pages/whats-inside.adoc

```diff
@@ -1,4 +1,4 @@
-= What's inside?
+= Concepts and Content
```

Comment:
@anarthal No, it is generally bad practice to make a headline a question - can be too easily misinterpreted. Perhaps just "Content", or even "Making Progress" (to align with Getting Started, and perhaps Advanced Topics - or similar).

Reply:
Yes. Removing the term "concept" and not making it a question are good ideas.

We need a mix of things that represent "what's inside":

- Contents
- Overview
- Introduction
- Exploring the Contents
- Understanding the Components
- Unveiling the Contents
- Insights and Breakdown
- Internal Features
- A Closer Look
- Quick Look
- ...

Reply:
@alandefreitas Exploring might work - captures what a new user is really doing.

Reply:
@alandefreitas @anarthal This looks decent in the local build - with Explore the Contents for whats-inside:

* xref:intro.adoc[]
* xref:whats-inside.adoc[]
* xref:getting-started.adoc[]

* Common Scenarios
** xref:reporting-issues.adoc[]
** xref:faq.adoc[]
** xref:testing-debugging.adoc[]

* Tutorials
** xref:boost-test-tutorial.adoc[]

* Advanced Scenarios
** xref:advanced-builds.adoc[]

* Community
** xref:bsl.adoc[]
** xref:release-process.adoc[] 
** xref:boost-history.adoc[]

* Reference
** xref:library-naming.adoc[]
** xref:header-organization-compilation.adoc[]

Reply:
Yes. The hierarchy is more well balanced.

I'm not sure reporting issues is a common scenario let alone the most common scenario because I've been a Boost user for over a decade and never reported an issue. 

In any case, we keep working on the structure as we come up with content. So we don't have to get stuck on this.